curator/README.md

```diff
@@ -108,21 +113,10 @@ configuration in the deployed configmap:
 
     $ oc edit configmap/logging-curator
 
-If this does not redeploy automatically, redeploy manually:
-
-    $ oc deploy --latest logging-curator
-
 For scripted deployments, do this:
 
     $ create /path/to/mycuratorconfig.yaml
     $ oc delete configmap logging-curator ; sleep 1
     $ oc create configmap logging-curator --from-file=config.yaml=/path/to/mycuratorconfig.yaml ; sleep 1
 
 Then redeploy as above.
-
-You can also specify default values for the run hour, run minute, and age in
-days of the indices when processing the curator template.  Use
-`CURATOR_RUN_HOUR` and `CURATOR_RUN_MINUTE` to set the default runhour and
-runminute, `CURATOR_RUN_TIMEZONE` to set the run timezone, and use
-`CURATOR_DEFAULT_DAYS` to set the default index age in days. These are only
-used if not specified in the config file.
```

Comment:
Do we still support these env. vars?  If not, why?

Reply:
Is there some way we can "translate" these into the cronjob settings?

Reply:
They're not used by curator or the associated scripts anymore.

Instead the playbook uses them to generate the cronjob k8s object:
https://github.com/openshift/openshift-ansible/pull/7047/files#diff-f4c6ebdf7da2e4a833073d1f89e08221R164